src/styles/blocks/header.scss

```
@@ -0,0 +1,64 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+.header-menu {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+
+  &__image {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.nav__list {
+  @extend %resetMarginPadding ;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+
+  &-item {
+    margin-right: 20px;
+    list-style: none;
+    position: relative;
+    text-transform: uppercase;
+    text-decoration: none;
+
+     &:last-child {
+       margin-right: 0;
+    }
```

use `:not()`, then you can remove `margin-right: 20px;` above
```suggestion
     &:not(:last-child) {
       margin-right: 20px;
    }
```